app/screens/home/channel_list/categories_list/header/index.ts

```diff
@@ -21,6 +21,12 @@ const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
 
     const currentUser = observeCurrentUser(database);
 
+    const config = observeConfig(database);
+
+    const enableOpenServer = config.pipe(
+        switchMap((t) => of$(t?.EnableOpenServer === 'true')),
+    );
```

Comment:
you can use `observeConfigBooleanValue` instead

Reply:
Done.